lib/config/error-codes/index.js

```diff
@@ -0,0 +1,70 @@
+/*
+ * Kuzzle, a backend software, self-hostable and ready to use
+ * to power modern apps
+ *
+ * Copyright 2015-2018 Kuzzle
+ * mailto: support AT kuzzle.io
+ * website: http://kuzzle.io
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const errors = require('kuzzle-common-objects').errors;
+const api = require('./api');
+const assert = require('assert');
+
+/** Check the format of the error-codes config files
+ *  @param {object} - error config file domain
+ */
+
+function checkErrorCodes(errorCodes) {
+
+  for (const domain of Object.keys(errorCodes)) {
```

Comment:
```suggestion
  for (const domain in errorCodes) {
```
More readable and efficient:
```
for..of x 17,096,741 ops/sec ±1.14% (86 runs sampled)
for..in x 29,050,072 ops/sec ±1.32% (86 runs sampled)
Fastest is for..in
```